app/models/grda_warehouse/file.rb

```diff
@@ -9,7 +9,6 @@ class File < GrdaWarehouseBase
     acts_as_paranoid
     belongs_to :user, optional: true
 
-    mount_uploader :file, FileUploader # Tells rails to use this uploader for this model.
-
+    # mount_uploader :file, FileUploader # Tells rails to use this uploader for this model.
```

Comment:
We could remove this, or just note that it has been commented out for now in favor of adding it where appropriate in sub-classes.

Reply:
Removed